.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug
+about: "Report a confirmed bug. For unconfirmed bugs please
+ visit https://discuss.elastic.co/c/logstash"
+labels: "bug,status:needs-triage"
+
+---
+<!--
+GitHub is reserved for bug reports and feature requests; it is not the place
+for general questions. If you have a question or an unconfirmed bug , please
+visit the [forums](https://discuss.elastic.co/c/logstash).  Please also
+check your OS is [supported](https://www.elastic.co/support/matrix#show_os).
+If it is not, the issue is likely to be closed.
+
+For security vulnerabilities please only send reports to security@elastic.co.
+See https://www.elastic.co/community/security for more information.
+
+Please fill in the following details to help us reproduce the bug:
+-->
+
+**Logstash version** (`bin/logstash --version`):
```

Comment:
perhaps a couple more useful bits that almost always come up:
 - how was Logstash installed? (examples: built from source, installed with system package manager, expanded from a tar or zip archive)
 - how is Logstash being run when it exhibits symptoms (e.g. under a service manager, via command line)